mozdownload/cli.py

```diff
@@ -0,0 +1,240 @@
+                   'try': scraper.TryScraper,
+                   }
+
+    usage = 'usage: %prog [options]'
+    parser = OptionParser(usage=usage, description=__doc__)
+    parser.add_option('--application', '-a',
+                      dest='application',
+                      choices=scraper.APPLICATIONS,
+                      default='firefox',
+                      metavar='APPLICATION',
+                      help='The name of the application to download, '
+                           'default: "%default"')
+    parser.add_option('--base_url',
+                      dest='base_url',
+                      default=scraper.BASE_URL,
+                      metavar='BASE_URL',
+                      help='The base url to be used, '
+                           'default: "%default"')
+    parser.add_option('--destination', '-d',
+                      dest='destination',
+                      default=os.getcwd(),
+                      metavar='DESTINATION',
+                      help='Directory or file name to download the '
+                           'file to, default: current working directory')
+    parser.add_option('--build-number',
+                      dest='build_number',
+                      type="int",
+                      metavar='BUILD_NUMBER',
+                      help='Number of the build (for candidate, daily, '
+                           'and tinderbox builds)')
+    parser.add_option('--locale', '-l',
+                      dest='locale',
+                      metavar='LOCALE',
+                      help='Locale of the application, default: "en-US or '
+                           'multi"')
+    parser.add_option('--platform', '-p',
+                      dest='platform',
+                      choices=scraper.PLATFORM_FRAGMENTS.keys(),
+                      metavar='PLATFORM',
+                      help='Platform of the application')
+    parser.add_option('--stub',
+                      dest='is_stub_installer',
+                      action='store_true',
+                      help='Stub installer. '
+                           'Only applicable to Windows builds.')
+    parser.add_option('--type', '-t',
+                      dest='type',
+                      choices=build_types.keys(),
+                      default='release',
+                      metavar='BUILD_TYPE',
+                      help='Type of build to download, default: "%default"')
+    parser.add_option('--url',
+                      dest='url',
+                      metavar='URL',
+                      help='URL to download. Note: Reserved characters (such '
+                           'as &) must be escaped or put in quotes otherwise '
+                           'CLI output may be abnormal.')
+    parser.add_option('--version', '-v',
+                      dest='version',
+                      metavar='VERSION',
+                      help='Version of the application to be used by release\
```

Comment:
That's something I would also do in the argparse PR given that I would have to touch those lines anyway.